data/json/monsters/zed_misc.json

```diff
@@ -442,6 +442,57 @@
       "FILTHY"
     ]
   },
+  {
+    "id": "mon_zombie_nemesis",
+    "type": "MONSTER",
+    "name": { "str": "relentless hulk" },
+    "description": "A hulking zombie swollen to the size of six men.  Its inky black eyes are locked on you.  No matter how far you run, it always seems to find you.",
+    "default_faction": "zombie",
+    "bodytype": "human",
+    "species": [ "ZOMBIE", "HUMAN" ],
+    "diff": 5,
+    "volume": "875000 ml",
```

Comment:
It seems to me that you need to balance volume/weight balance as for now the density of a monster is <0.25, and that's just weird. Not enough to float, but that's barely different from aerogel.

Reply:
Good catch, I can make some adjustments. Worth noting these values are copied from the skeletal juggernaut, where they would make a little more sense if it was a doot doot-style cartoon skeleton, but considering it's stylized as bones and flesh, that might warrant a change as well.